libs/common-utils/src/tokens.ts

```diff
@@ -14,3 +14,16 @@ export function isNativeAddress(tokenAddress: string, chainId: ChainId): boolean
 
   return native && doesTokenMatchSymbolOrAddress(native, tokenAddressLower)
 }
+
+export function getTokenAddressKey(address: string): string {
+  return address.toLowerCase()
+}
+
+export interface TokenIdentifier {
+  address: string
+  chainId: number
+}
+
+export function getTokenId(token: TokenIdentifier): string {
```

Comment:
nitpick: would be nice to define a type for that like export type TokenId = `${number}:${string}`
for better understanding